infra/modules/providers/azure/ad-application/output.tf

```diff
@@ -0,0 +1,14 @@
+output "application_id" {
```

Comment:
Because we're outputting a map of azuread_application.auth.name to the azuread_application.auth.application_id, do we really need this also? I prefer the mapped version since we're possibly creating more than one app registration and it provides flexibility of more context.

Reply:
This was removed.